src/set_theory/cardinal/cofinality.lean

```diff
@@ -1078,3 +1078,41 @@ begin
 end
 
 end cardinal
+
+section omega_1
+
+namespace ordinal
+open cardinal
+open_locale ordinal cardinal
+
+lemma sup_sequence_lt_omega_1 (o : ℕ → ordinal.{u}) (ho : ∀ n, o n < ω₁) :
```

Comment:
Might be nice generalizing this to countable types.

Reply:
Yes! **Question**: Once stated for countable types, would it work automatically in the use case? Or one should have this particular one stated as well? 

Actually there are plenty of generalizations; the other I can think of is replacing `ω₁` by any ordinal of uncountable cofinality. 

As a matter of fact, this is a particular case of König's theorem, where instead of `ℕ` you have any type of cardinality less than the cofinality of the bound (that would be the new `ho`), and the conclusion follows as well. That is another place where there is plenty of API to work on.

Reply:
If you state it for `{α} [countable α] (o : α → ordinal.{u})` it should apply to `ℕ` since there's an instance [nat.countable](https://leanprover-community.github.io/mathlib_docs/data/countable/defs.html#nat.countable).